aten/src/ATen/core/op_registration/op_registration_test.cpp

```diff
@@ -789,8 +789,8 @@ TEST(OperatorRegistrationTest, whenRegisteringBackendFallbackKernelAndCatchallKe
 
   called = false;
   auto stack = callOp(*op, dummyTensor(c10::DispatchKey::CPU), "hello ");
-  EXPECT_FALSE(called);
```

Comment:
Does it make sense to move the modified tests over to NewOperatorRegistrationTest?

Reply:
Yep! I'll move them in this PR!